.changeset/config.json

```diff
@@ -5,7 +5,7 @@
   "fixed": [],
   "linked": [],
   "access": "public",
-  "baseBranch": "main",
+  "baseBranch": "origin/dev",
```

Comment:
what's this change for?

Reply:
It was not letting me write the changeset. I had to change it in order for `yarn changeset` to work

Reply:
it's weird that no one else has had this problem.

lets revert this for now and keep an eye out if it happens again in your next PR

Reply:
Gotcha